packages/docs/scripts/generate-docs.js

```diff
@@ -0,0 +1,129 @@
+            'heading-depth': 1,
+        });
+
+        // if there's markdown, do stuff
+        if (markdown && markdown.length > 0) {
+            // get the package ID from the file path
+            const matchingExpression = filePath.match(
+                /\/packages\/([\s\S]*?)\/src\/charts\/([\s\S]*?)\//i
+            );
+            const packageId = matchingExpression[1];
+            const chartId = matchingExpression[2];
+            const chartName =
+                chartId.charAt(0).toUpperCase() + chartId.slice(1);
+
+            if (chartName !== processingPackageName) {
+                processingPackageName = chartName;
+                console.log(
+                    `-= Processing the ${chartName} chart from the ${packageId} package =-`
+                );
+            }
+
+            // get the sub-folder structure relative to /src/
+            let subPath = path
+                .dirname(filePath)
+                .match(/\/src\/charts\/([\s\S]*?)$/i)
+                ? `${
+                      path
+                          .dirname(filePath)
+                          .match(/\/src\/charts\/([\s\S]*?)$/i)[1]
+                  }`
+                : '';
+
+            // Get each part of the path, filtering out empty path items
+            const subPathArray = subPath.split('/');
+
+            // get file name sans extension
+            let fileName = path.basename(filePath, '.js');
+
+            // if the file name is index, but is not the package index
+            if (subPathArray.length > 0 && fileName === 'index') {
+                // change file name to the parent folder name
+                fileName = subPathArray[subPathArray.length - 1];
+
+                // remove the parent folder from the path
+                subPathArray.pop();
+                subPath = subPathArray.join('/');
+            }
+
+            const writeDir = path.join(__dirname, `../docs/API/`);
+
+            // check if the directory exists
+            if (!fs.existsSync(writeDir)) {
+                // create the directory
+                fs.mkdirSync(writeDir, { recursive: true });
+            }
+
+            // write the markdown file
+            fs.writeFileSync(`${writeDir}/${fileName}.md`, markdown);
+        }
+    });
+
+    // Let the user know what step we're on
+    console.log('\u001B[32m', '✔️  Docs generated', '\u001B[0m');
+
+    // Let the user know what step we're on
+    // console.log('-= Updating sidebars.js =-');
+
+    // write sidebar file
+    // fs.writeFileSync(
+    //     path.join(__dirname, '../sidebars.js'),
+    //     `module.exports = ${JSON.stringify(sidebars, null, '  ')}`,
+    //     'utf8'
+    // );
+
+    // run prettier on the output json
+    // execSync(`eslint --fix ${path.join(__dirname, '../docs/sidebars.js')}`);
+
+    // Let the user know what step we're on
+    // console.log('\u001B[32m', '✔️  sidebars.js updated', '\u001B[0m');
```

Comment:
I don't think this part is necessary anymore.

Reply:
True, removing.